cmd/create-repo.go

```diff
@@ -123,7 +141,9 @@ func init() {
 	createrepoCmd.Flags().String("repo", "", "Use repository defined in configuration.")
 
 	createrepoCmd.Flags().String("tree-compression", "none", "Compression alg: none, gzip")
-	createrepoCmd.Flags().String("tree-path", installer.TREE_TARBALL, "Repository tree filename")
+	createrepoCmd.Flags().String("tree-name", installer.TREE_TARBALL, "Repository tree filename")
```

Comment:
`tree-name` doesn't tell much about the file, but looks like the "tree name" .. maybe better `tree-filename` instead ?

Reply:
so similar for `meta-filename`?